cpp/modmesh/view/wrap_view.cpp

```diff
@@ -95,9 +154,9 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapRLine
         (*this)
             .def(
                 py::init(
-                    [](float x0, float y0, float z0, float x1, float y1, float z1, uint8_t color_r, uint8_t color_g, uint8_t color_b)
+                    [](R3DWidget & w, float x0, float y0, float z0, float x1, float y1, float z1, uint8_t color_r, uint8_t color_g, uint8_t color_b)
```

Comment:
Simply make it compiled.  I did not test it and would guess it's not working anymore.